tests/unit/src/test/scala/tests/NewFilesLspSuite.scala

```diff
@@ -102,7 +103,7 @@ class NewFilesLspSuite extends BaseLspSuite("new-files") {
 
   private def indent = "  "
 
-  private def check(testName: String)(
+  private def check(testName: TestOptions)(
```

Comment:
just out of curiosity:
1) why?
2) Is there implicit conversion happening? I see, there is `TestOptionsConversions` at munit, but it does not look imported and available...

Reply:
I can revert, but this enables adding tags and ignoring.

Reply:
We also do this in other places, so we probably don't have to revert.